.travis.yml

```diff
@@ -1,11 +1,20 @@
 language: java
 
-install: mvn install -DskipTests=true -Dgpg.skip=true
+before_install:
+  - nvm install $NODE_VERSION
+  - curl -s "https://get.sdkman.io" | bash
+  - source "$HOME/.sdkman/bin/sdkman-init.sh"
+  - sdk install java 17.0.5-amzn
```

Comment:
Are we able to use Oracle's Java 17?

Reply:
Changed in [90aae3c](https://github.com/bullhorn/sdk-rest/pull/405/commits/90aae3c585b4c7c1d9f5697a120602c51d4c5e7b). We'll see how the Travis build goes.